.claude/skills/cfn-docker-loop-orchestration/orchestrate.sh

```
@@ -98,6 +102,72 @@ Examples:
+    # SECURITY FIX #2: Command injection prevention - strict alphanumeric whitelist
+    # Allows ONLY: letters, numbers, dash, underscore, space, comma, period, colon
+    local sanitized=$(echo "$input" | tr -cd '[:alnum:] _,.:-')
+
+    # Length bounds check
+    if [ ${#input} -gt "$max_length" ]; then
+        log_error "Input exceeds maximum length ($max_length): ${#input}"
+        return 1
+    fi
+
+    # Reject if input contains shell metacharacters: $, `, ;, |, &, >, <, (, ), {, }, [, ], \, ", ', =
+    if [[ "$input" =~ (\$|`|;|\||&|>|<|\(|\)|\{|\}|\[|\]|\\|\"|\'|=) ]]; then
```

Shellcheck (0.11.0)

[error] Couldn't parse this regex grouping. Fix to allow more checks.

(SC1073)